include/simdjson/dom/serialization.h

```diff
@@ -5,7 +5,7 @@
 #include "simdjson/dom/element.h"
 #include "simdjson/dom/object.h"
 
-#include <vector>
+#include <utility>
```

Comment:
This is fine, but by removing `vector` as an included header, you will need to include it in our tests so that the project builds. I am not sure why you are removing vector.